.vscode/settings.json

```diff
@@ -6,6 +6,10 @@
     // reformatting of code.
     "editor.formatOnSave": false,
```

Comment:
Do we need these lines then?

Reply:
nvm, we probably don't want to be autoformatting anything in this repo

Reply:
Vs-code doesn't respect the default rules of current .settings for the workspace scope, if you have especified some different behavior for ["dart"] in your global setup scope.

Added this lines to ensure respect for the original rules, avoiding auto formatting.